lib/infinispan.js

```diff
@@ -228,6 +228,14 @@
             .then(function() {
               logger.debugf('Started Infinispan %s', client);
               return client; // return client
+            }, function(reason){
```

Comment:
`catch()` is preferred and consistent with the rest of the codebase.

```
.then()
.catch(function(error) { ... })
```

Reply:
Changed according to comment.